db/models.py

```diff
@@ -1 +1,47 @@
-from django.db import models
+from django.db import models as m
+
+
+class Race(m.Model):
+    races = (
+        ("E", "Elf"),
+        ("D", "Dwarf"),
+        ("H", "Human"),
+        ("O", "Ork")
+    )
+
+    name = m.CharField(max_length=255, unique=True, choices=races)
+    description = m.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return f"The race is:{self.name}"
+
+
+class Skill(m.Model):
+    name = m.CharField(max_length=255, unique=True)
+    bonus = m.CharField(max_length=255)
+    race = m.ForeignKey(Race, on_delete=m.PROTECT)
```

Comment:
`PROTECT` is not the best option here. Should you really be unable to delete a race without manually deleting all the skills?